README.md

```diff
@@ -78,6 +79,60 @@ then do an HTTP POST to the following URL:
     }
 ```
 
+### Sending custom Cloud Connector events via REST API and RabbitMQ
+
+The Cloud Connector event is a special case where you can **_override_** the `inBoundVariables` and/or `outBoundVariables` attributes.
+
+The default Cloud Connector event looks like:
+
+```
+{
+    "appName": "default-app",
+    "appVersion": "",
+    "serviceName": "rb-my-app",
+    "serviceFullName": "rb-my-app",
+    "serviceType": "runtime-bundle",
+    "serviceVersion": "",
+    "integrationContext": {
+        "id": "1c7b99de-fa50-11e8-9b6d-8efb5254abd4",
+        "outBoundVariables": {},
+        "processInstanceId": "1c7b72c8-fa50-11e8-9b6d-8efb5254abd4",
+        "processDefinitionId": "ConnectorProcess:1:41f6f25f-fa0a-11e8-9b6d-8efb5254abd4",
+        "activityElementId": "sid-CDFE7219-4627-43E9-8CA8-866CC38EBA94",
+        "connectorType": "Example Connector",
+        "inBoundVariables": {
+            "firstName": "Paulo",
+            "lastName": "Silva",
+            "age": 25
+        }
+    }
+}
+```
+
+To override the default event, first run the application with the following settings:
+
+    mvn spring-boot:run -Dspring.profiles.active=rabbitMQ -Dgenerator.startSendAtStartup=false -Dgenerator.eventCategory=CLOUD_CONNECTOR_EVENT -Dmessaging.to.rabbitmq.camelRoute.destinationName="Example Connector"
+
+then do an HTTP POST to the following URL:
+
+    http://<host>:<port>/alfresco/mock/connector-event
+
+Request payload example:
+
+```
+{
+    "inBoundVariables": {
+      "nodeId": "bd285b4f-9ee4-11e8-9079-0242ac118745",
+      "properties": {
+        "cm:title": "Node Title",
+        "cm:description": "Node Description"
+      }
+    }
+}
+```
+
+**Note:** `inBoundVariables` and `outBoundVariables` are of type `Map<String, Object>` so you can add whatever you require withing the JSON body.
```

Comment:
I think this should read "within the JSON body."

Reply:
Good catch!

Reply:
Done.